tdvt/test/tool_test/test_generation/full_map_to_calcs.json

```diff
@@ -0,0 +1,137 @@
+{
+    "k": {
+        "type": "VARCHAR",
+        "data_shape": "no_empties_no_nulls",
+        "alts": "False"
```

Comment:
Just an idea: these could be their own boolean properties: empties, negatives, nulls

Reply:
That sounds fine to me - so we would wind up with an object like this:

``` json
{
      "type": "FLOAT",
      "empties": "True",
      "negatives": "False",
      "nulls": "False"
},
```
much easier to understand and type

Reply:
Changed this in [3cd7571](https://github.com/tableau/connector-plugin-sdk/pull/1101/commits/3cd75712d71b12912e47641401ddf32959c37ddb). Also sorted keys alphabetically.